docs/getting-started-guides/gce.md

```diff
@@ -71,9 +75,7 @@ cluster/kubectl.sh delete rc my-nginx
 
 ### Running a container (more complete version)
 
-
-Assuming you've run `hack/dev-build-and-up.sh` and `hack/build-go.sh`, you
-can create a pod like this:
+Note: this requires checking out the repo to get `docs/`.
```

Comment:
Binary releases now (have they always?) contain the docs/ directory, so we probably don't need this line.